client/src/components/leftSidebar/topLeftLogoAndTitle.js

```diff
@@ -107,9 +107,9 @@ class LeftSideBar extends React.Component {
               userinfo,
             }}
           />
-          {!userinfo.is_authenticated ? (
+          {!userinfo.is_authenticated && (
```

Comment:
🎉  I think we can even get rid of the ()

```suggestion
          {!userinfo.is_authenticated && <AuthButtons auth={auth} userinfo={userinfo} />}
```

Reply:
Ah good point! In this case, since `prettier` wraps code when the line is too long, it automatically does the following instead!

```ts
{!userinfo.is_authenticated && (
  <AuthButtons auth={auth} userinfo={userinfo} />
)}
```

Reply:
ooh nvm then!